zigpy_deconz/types.py

```diff
@@ -1,5 +1,7 @@
 import enum
 
+import zigpy.types as t
```

Comment:
I would prefer to not have any zigpy dependencies except for `zigbee.application`. What do you think?

Reply:
Fine by me, but I'd like to have NWK class defined with Hex representation like it is in zigpy. Such a small thing, but makes tons of difference when sipping through debug logs.

Reply:
Removed Zigpy types dependency.